src/sourabh/testCase11/TestCase11.java

```diff
@@ -0,0 +1,88 @@
+/*
+Test case - 11: 16th Jun'2021
+
+program 1: How many Depts we have.
+ans : 4
+
+program 2 : how many managers we have.
+ans : 3
+
+program 3: how many employees we have.
+ans : 8
+		 
+ */
+
+package sourabh.testCase11;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class TestCase11 {
+	static int getDeptCount(WebDriver driver) {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method